src/Ren/Data/Expression/Literal.elm

```diff
@@ -3,6 +3,7 @@ module Ren.Data.Expression.Literal exposing
     , coerceToNumber, coerceToInteger, coerceToString
     , decoder, primitiveDecoder
     , parser, primitiveParser
+    , TemplateSegment(..)
```

Comment:
I'm not sure if this is the correct way to export this, as it clutters the `Literal `namespace with `Text` and `Expr`.
Unfortunately I couldn't get it to work without exporting `(..)` (probably due to my lacking Elm knowledge).